File: app/src/main/java/com/otitan/model/ResourceModel.java
package com.otitan.model;

import java.io.Serializable;

/**
 * 资源管理
 */
public class ResourceModel implements Serializable {

    /**
     * 林地面积
     */
    public class LDMJModel implements Serializable {
        //地区编码
        private String Code;
        //地区名称
        private String Name;
        //林地总面积
        private String Area;
        //乔木林地
        private String QMLD;
        //竹林地
        private String ZLD;
        //疏林地
        private String SLD;
        //	特殊灌木林地
        private String TSGMLD;
        //	一般灌木林地
        private String YBGMLD;
        //	未成林造林地
        private String WCLZLD;
        //宜林地
        private String YLD;
        //	其它
        private String Other;

        public String getCode() {
            return Code;
        }

        public void setCode(String code) {
            Code = code;
        }

        public String getName() {
            return Name;
        }

        public void setName(String name) {
            Name = name;
        }

        public String getArea() {
            return Area;
        }

        public void setArea(String area) {
            Area = area;
        }

        public String getQMLD() {
            return QMLD;
        }

        public void setQMLD(String QMLD) {
            this.QMLD = QMLD;
        }

        public String getZLD() {
            return ZLD;
        }

        public void setZLD(String ZLD) {
            this.ZLD = ZLD;
        }

        public String getSLD() {
            return SLD;
        }

        public void setSLD(String SLD) {
            this.SLD = SLD;
        }

        public String getTSGMLD() {
            return TSGMLD;
        }

        public void setTSGMLD(String TSGMLD) {
            this.TSGMLD = TSGMLD;
        }

        public String getYBGMLD() {
            return YBGMLD;
        }

        public void setYBGMLD(String YBGMLD) {
            this.YBGMLD = YBGMLD;
        }

        public String getWCLZLD() {
            return WCLZLD;
        }

        public void setWCLZLD(String WCLZLD) {
            this.WCLZLD = WCLZLD;
        }

        public String getYLD() {
            return YLD;
        }

        public void setYLD(String YLD) {
            this.YLD = YLD;
        }

        public String getOther() {
            return Other;
        }

        public void setOther(String other) {
            Other = other;
        }
    }

    /**
     * 森林面积
     */
    public class SLMJModel implements Serializable {
        //地区编码
        private String Code;
        //地区名称
        private String Name;
        //林地总面积
        private String Area;

        public String getCode() {
            return Code;
        }

        public void setCode(String code) {
            Code = code;
        }

        public String getName() {
            return Name;
        }

        public void setName(String name) {
            Name = name;
        }

        public String getArea() {
            return Area;
        }

        public void setArea(String area) {
            Area = area;
        }
    }

    /**
     * 公益林面积
     */
    public class GYLMJModel implements Serializable {
        //地区编码
        private String Code;
        //地区名称
        private String Name;
        //国家级公益林
        private String CountryArea;
        //地方公益林
        private String LocalArea;
        //合计
        private String Area;

        public String getCode() {
            return Code;
        }

        public void setCode(String code) {
            Code = code;
        }

        public String getName() {
            return Name;
        }

        public void setName(String name) {
            Name = name;
        }

        public String getCountryArea() {
            return CountryArea;
        }

        public void setCountryArea(String countryArea) {
            CountryArea = countryArea;
        }

        public String getLocalArea() {
            return LocalArea;
        }

        public void setLocalArea(String localArea) {
            LocalArea = localArea;
        }

        public String getArea() {
            return Area;
        }

        public void setArea(String area) {
            Area = area;
        }
    }

    /**
     * 森林覆盖率
     */
    public class SLFGLModel implements Serializable {
        //地区编码
        private String Code;
        //地区名称
        private String Name;
        //森林覆盖率
        private double Area;

        public String getCode() {
            return Code;
        }

        public void setCode(String code) {
            Code = code;
        }

        public String getName() {
            return Name;
        }

        public void setName(String name) {
            Name = name;
        }

        public double getArea() {
            return Area;
        }

        public void setArea(double area) {
            Area = area;
        }
    }

    /**
     *  活立木蓄积
     */
    public class HLMXJModel implements Serializable {
        //地区编码
        private String Code;
        //地区名称
        private String Name;
        //活立木蓄积
        private double Area;

        public String getCode() {
            return Code;
        }

        public void setCode(String code) {
            Code = code;
        }

        public String getName() {
            return Name;
        }

        public void setName(String name) {
            Name = name;
        }

        public double getArea() {
            return Area;
        }

        public void setArea(double area) {
            Area = area;
        }
    }
}

File: app/src/main/java/com/otitan/model/ResourceModel.kt
package com.otitan.model

import java.io.Serializable

/**
 * 资源管理
 */
class ResourceModel : Serializable {

    /**
     * 林地面积
     */
    inner class LDMJModel : Serializable {
        //地区编码
        var Code: String? = null
        //地区名称
        var Name: String? = null
        //林地总面积  林地面积(数据管理)
        var Area: String? = null
        //乔木林地
        var QMLD: String? = null
        //竹林地
        var ZLD: String? = null
        //疏林地
        var SLD: String? = null
        //	特殊灌木林地
        var TSGMLD: String? = null
        //	一般灌木林地
        var YBGMLD: String? = null
        //	未成林造林地
        var WCLZLD: String? = null
        //宜林地
        var YLD: String? = null
        //	其它
        var Other: String? = null
        //年份(数据管理)
        var Year: String? = null
        //校正面积(数据管理)
        var CorrectionArea: String? = null
    }

    /**
     * 森林面积
     */
    inner class SLMJModel : Serializable {
        //地区编码
        var Code: String? = null
        //地区名称
        var Name: String? = null
        //林地总面积
        var Area: String? = null
        //年份(数据管理)
        var Year: String? = null
        //森林面积(数据管理)
        var TotalArea: String? = null
        //校正面积(数据管理)
        var TotalCorrectionArea: String? = null
    }

    /**
     * 公益林面积
     */
    inner class GYLMJModel : Serializable {
        //地区编码
        var Code: String? = null
        //地区名称
        var Name: String? = null
        //国家级公益林
        var CountryArea: String? = null
        //地方公益林
        var LocalArea: String? = null
        //合计
        var Area: String? = null
        //年份(数据管理)
        var Year: String? = null
        //公益林总面积(数据管理)
        var TotalArea: String? = null
    }

    /**
     * 森林覆盖率
     */
    inner class SLFGLModel : Serializable {
        //地区编码
        var Code: String? = null
        //地区名称
        var Name: String? = null
        //森林覆盖率
        var Area: Double = 0.0
        //年份(数据管理)
        var Year: String? = null
        //森林覆盖率(数据管理)
        var Per: String? = null
        //校正森林覆盖率(数据管理)
        var CorrectPer: String? = null

    }

    /**
     * 活立木蓄积
     */
    inner class HLMXJModel : Serializable {
        //地区编码
        var Code: String? = null
        //地区名称
        var Name: String? = null
        //活立木蓄积
        var Area: Double = 0.0
        //活立木蓄积(数据管理)
        var Standing: Double = 0.0
        //校正活立木蓄积(数据管理)
        var CorrectStanding: Double = 0.0
    }
}
